core/src/test/scala/integration/kafka/coordinator/transaction/ProducerIntegrationTest.scala

```diff
@@ -182,9 +182,9 @@ class ProducerIntegrationTest {
 
   private def verifyUniqueIds(clusterInstance: ClusterInstance): Unit = {
     // Request enough PIDs from each broker to ensure each broker generates two blocks
-    val ids = clusterInstance.brokerSocketServers().stream().flatMap( broker => {
+    val ids = clusterInstance.brokers().values().stream().flatMap( broker => {
```

Comment:
nit: redundant space
```suggestion
    val ids = clusterInstance.brokers().values().stream().flatMap(broker => {
```